### DISCUSSION TITLE
启用可执行手势的任意无障碍应用后 XBOX 手柄部分按键无响应

### DISCUSSION THREAD

**Discussion by ztelliot** [gkd-kit/gkd#788] [Announcements]
### 日志文件

[log-1732332812376.zip](https://github.com/user-attachments/files/17878296/log-1732332812376.zip)

### BUG描述(文字/截图/视频)

当启用 GKD 的无障碍服务后，XBOX 手柄大部分按键不可用

### 期望行为(文字/截图/视频)

下面是关闭 GKD 无障碍服务后的通过 steam link 测试手柄的情况

https://github.com/user-attachments/assets/177794fc-56e7-440a-8146-0d789dd6773a

### 实际行为(文字/截图/视频)

下面是打开 GKD 无障碍服务后的手柄测试情况，可以看到，方向键和扳机键均无响应

https://github.com/user-attachments/assets/218d8dbf-7826-478d-a8df-137415a48012

**Comment by lisonge** [comment]
关闭 gkd，开启其它无障碍应用是否会出现此情况？

**Comment by ztelliot** [comment]
其他无障碍应用没有出现这种情况，我的 Bitwarden 和 连接到 Windows 的无障碍在测试中是保持开启状态的

**Comment by lisonge** [comment]
关闭 `高级设置-前台悬浮窗` 呢

**Comment by lisonge** [comment]
Bitwarden 没有声明派发手势的功能

<https://github.com/bitwarden/android/blob/main/app/src/main/res/xml/accessibility_service.xml>

你换一个其他自定义点击应用看看是否会出现此情况？

**Comment by ztelliot** [comment]
> 关闭 `高级设置-前台悬浮窗` 呢

关闭之后问题仍然存在

> 你换一个其他自定义点击应用看看是否会出现此情况？

试了下李跳跳也会导致一样的情况

**Reply by lisonge** [reply]
https://github.com/gkd-kit/gkd/blob/f88bc1fb3c11bccdae932a97345127c42421ff5a/app/src/main/res/xml/ab_desc.xml#L7

应该是这个导致的，点击屏幕需要这个功能

---

另外一个解决方法是授予 写入安全设置权限 后可以通过通知面板 快捷图标 关闭/开启 无障碍

**Reply by ztelliot** [reply]
🫡感谢解答
